### ISSUE TITLE
tried editing an experiment - and got spin wheel of death

### ISSUE THREAD

**Issue by shell1** [mozilla/experimenter#2168]
was messing around on https://stage.experimenter.nonprod.dataops.mozgcp.net/experiments/experiment-name-v23/ and couldn't see the design - just wheel spinning.

![image](https://user-images.githubusercontent.com/8192232/72309846-fa137580-3634-11ea-9c77-9bfbd81d9d33.png)

**Comment by tiftran** [comment]
Hi @shell1 
There was a bug introduced a long time ago ref: https://github.com/mozilla/experimenter/issues/1852
In summary experiments created before a certain time do not have a branch that is marked as the control branch, which experimenter needs or it shows the spinning wheel of death :(

The experiment you're referring to was created in a time where the bug existed. Through the admin console we can see neither branches have the "is_control" checkbox checked.
![Screen Shot 2020-01-13 at 7 22 06 PM](https://user-images.githubusercontent.com/25379936/72311978-6c875400-363b-11ea-9e85-cee2722db554.png)

I've went into the admin and checked the control branch's "is_control" checkbox and I believe you shouldn't see the spinning wheel of death now. This shouldn't be able to occur in prod.... since we went and fixed the experiments by hand through the admin console...
